app/controllers/controllershelpers/HomeCardGenerator.scala

```diff
@@ -95,23 +96,24 @@ class HomeCardGenerator @Inject() (
       }
     }
 
-  def getSaAndItsaMergeCard()(implicit
+  def getSaAndItsaMergeCardOrSaWithoutUtrCard()(implicit
     messages: Messages,
     request: UserRequest[_]
-  ): Option[HtmlFormat.Appendable] =
-    if (
-      request.trustedHelper.isEmpty &&
-      (enrolmentsHelper.itsaEnrolmentStatus(request.enrolments).isDefined || request.isSa)
-    ) {
-      Some(
-        saAndItsaMergeView(
-          (current.currentYear + 1).toString,
-          enrolmentsHelper.itsaEnrolmentStatus(request.enrolments).isDefined
-        )
-      )
-    } else {
-      None
+  ): Option[HtmlFormat.Appendable] = {
+    val isItsaEnrolled = enrolmentsHelper.itsaEnrolmentStatus(request.enrolments).isDefined
+    val saView         = saAndItsaMergeView(
+      (current.currentYear + 1).toString,
+      isItsaEnrolled
+    )
```

Comment:
These could do with making a bit clearer. Maybe using if might be better than match? Also method name a bit long